lib/packageModules.js

```diff
@@ -139,7 +139,12 @@ module.exports = {
 
     // Copy artifacts to package location
     if (isIndividialPackaging.call(this)) {
-      _.forEach(functionNames, funcName => copyArtifactByName.call(this, funcName));
+      const nodeFunctionNames = _.filter(functionNames, funcName => {
+        const func = this.serverless.service.getFunction(funcName);
+        const runtime = func.runtime || this.serverless.service.provider.runtime || 'nodejs';
+        return runtime.match(/node/);
```

Comment:
Can you create a separate function to re-use this code from line 217?
For example, you can create that function just below the `getHandlerFile` function:

```js
    const isNodeRuntime = funcName => {
      const func = this.serverless.service.getFunction(funcName);
      const runtime = func.runtime || this.serverless.service.provider.runtime || 'nodejs';

      return runtime.match(/node/);
    };
```

And then, re-use it here (and when we build the `allEntryFunctions` const):
```suggestion
        return isNodeRuntime(funcName);
```

Reply:
Another idea might be to create a new function, that replaces `this.serverless.service.getAllFunctions()` calls and returns an already filtered list. It could be called something like `getAllNodeFunctions()`.

This would also prove useful to fix another issue I'm currently facing, where serverless-webpack still overrides the artifact path of Java based in packageModules.js:161

Reply:
Another option might be to add something like a `getAllNodeFunctions()` at a higher to replace `this.serverless.service.getAllFunctions()` calls with a filtered version. This could then also be used to provide similar logic in files like `validate.js`.